packages/cli/src/models/files/ZosNetworkFile.ts

```diff
@@ -59,6 +59,10 @@ interface ProviderInterface {
   address?: string;
 }
 
+interface ProxyAdminInterface {
```

Comment:
What would be the difference on using an `type` instead of an `interface` here, given it's a rather small object? Something like `type ProxyAdminInfo = { address: string }`. Moreover, we could have `type HasAddress = { address: string }` and use it for Package, Provider, and App as well.